bofire/strategies/predictives/predictive.py

```diff
@@ -83,6 +84,25 @@ def tell(
             self.set_experiments(experiments)
         else:
             self.add_experiments(experiments)
+        # we check here that the experiments do not have completely fixed columns
+        cleaned_experiments = (
+            self.domain.outputs.preprocess_experiments_all_valid_outputs(
+                experiments=experiments
+            )
+        )
+
+        fixed_nontasks = (
```

Comment:
Just one question: what is the purpose of a multitask optimization when your historical data is only for one task and you only want candidates in the future for the same task?

Reply:
Sorry. I wasn't clear in my formulation above. I have historical data of task 1 and I want to optimize task 2.